test/lib/code_corps_web/controllers/github_events_controller_test.exs

```diff
@@ -26,17 +26,6 @@ defmodule CodeCorpsWeb.GitHubEventsControllerTest do
     |> put_req_header("x-github-delivery", id)
   end
 
-  @access_token "v1.1f699f1069f60xxx"
-  @expires_at Timex.now() |> Timex.shift(hours: 1) |> DateTime.to_iso8601()
-  @access_token_create_response %{"token" => @access_token, "expires_at" => @expires_at}
-
-  @installation_created load_event_fixture("installation_created")
-  @installation_repositories load_endpoint_fixture("installation_repositories")
-
-  @tag bypass: %{
-    "/installation/repositories" => {200, @installation_repositories},
-    "/installations/#{@installation_created["installation"]["id"]}/access_tokens" => {200, @access_token_create_response}
-  }
   test "responds with 200 for a supported event", %{conn: conn} do
```

Comment:
This is an example of simply taking away the `bypass` tag. The default `SuccessAPI` is enough to keep the tests passing.